todo_app/templates/index.html

```diff
@@ -2,29 +2,34 @@
 {% block title %}To-Do App{% endblock %}
 
 {% block content %}
-  <div class="jumbotron">
-    <h1 class="display-4">To-Do App</h1>
-    <p class="lead">Just another to-do app.</p>
-  </div>
+<div class="jumbotron">
+  <h1 class="display-4">To-Do App</h1>
+  <p class="lead">Just another to-do app.</p>
+</div>
 
-  <div class="row">
-    <div class="col-md-12">
-      <h2>Items</h2>
-      <ul class="list-group mb-4">
-        {% for item in items %} 
-        <li>{{ item.status }} - {{item.title}}</li> 
-        {% endfor %} 
-      </ul>
-    </div>
+<div class="row">
+  <div class="col-md-12">
+    <h2>Items</h2>
+    <ul class="list-group mb-4">
+      {% for to_do_item in to_do_items %}
+      <li>{{to_do_item.status}} - {{to_do_item.name}}
+        <a href="{{ url_for('completeItem',id=to_do_item.id) }}">Mark as done</a>
+      </li>
+      {% endfor %}
+      {% for done_item in done_items %}
+      <li>{{done_item.status}} - {{done_item.name}}</li>
```

Comment:
If this is deliberately the same as the "to do" items, then you don't need to repeat it.

E.g. you could loop through a single list of items and use an appropriate if statement:
```
<li>
    {{done_item.status}} - {{done_item.name}}
    {% if ... %}
        <a ...
    {% endif %}
</li>
```

Reducing duplication becomes more relevant if the item becomes more complicated e.g. with delete buttons etc.